lightstep/resource_dashboard.go

```diff
@@ -135,6 +143,9 @@ func displayTypeOptionsFromResourceData(opts map[string]interface{}) *schema.Set
 	f := func(i interface{}) int {
 		return 1
 	}
+	if opts["y_axis_scale"] == "log" && opts["y_axis_log_base"] == "" {
+		opts["y_axis_log_base"] = 10
```

Comment:
Won't this make the terraform diff always show a difference when the `y_axis_log_base` is not set? It will always be comparing `10` to the empty string, right?

Reply:
Do you mean the diff on apply? I'm seeing it on every apply anyway. Unless you're referring to something else. I'm a noob at using terraform. I could also try figuring out where it's being saved and do it there (I'm guessing the public AP)
<img width="451" alt="Screenshot 2023-08-17 at 5 49 44 PM" src="https://github.com/lightstep/terraform-provider-lightstep/assets/5385318/6feef497-5e71-4432-a56d-50af758d746a">

Reply:
yeah the diff on the apply. I suspect that it would always see a change because it compares what is in the terraform file with what comes back from the GET endpoint (that we then mutate here)

Reply:
What I'd expect is the the first time it's applied there is a diff, but after that there is no diff

Reply:
Yeah, oddly, I see a diff for display_type_options happening even without my changes.